.github/workflows/publish.yml

```diff
@@ -33,11 +33,11 @@ jobs:
       - name: Build the binaries
         if: startsWith(github.ref, 'refs/tags/')
         run: |
+          go install github.com/mitchellh/gox@latest
           ./dists/make-release.sh -v ${{ steps.tag_name.outputs.TAG_NAME }}
 
       - name: Release the new version
         if: startsWith(github.ref, 'refs/tags/')
         uses: softprops/action-gh-release@v1
         with:
-          body: Please refer to the [CHANGELOG.md](https://github.com/Scalingo/cli/blob/${{ steps.tag_name.outputs.TAG_NAME }}/CHANGELOG.md) file.
```

Comment:
Why would like to we want to remove it?

Reply:
Oh sorry, I should have explained it before assigning you the review. I want to test if, when the body is empty, it uses the GitHub feature of automatically generating the changelog (https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes)